01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -57,4 +57,4 @@ public function getName()
     public function celebrateBirthday() {
         echo 'Yay!';
     }
-} 
\ No newline at end of file
+}
```

Comment:
Well, interesting that the absence of newlines in **most** of the files did not seem to be of interest to anyone. When adapting a few files to the rest, however, there you come complaining. :)

Again, I'm on your side here&mdash;I just thought it a bit too proactive to change almost any file.